tfhe/src/core_crypto/algorithms/lwe_multi_bit_programmable_bootstrapping.rs

```diff
@@ -61,16 +61,14 @@ pub fn prepare_multi_bit_ggsw_mem_optimized<
                 monomial_degree.wrapping_add(selection_bit.wrapping_mul(mask_element));
         }
 
-        let switched_degree = fast_pbs_modulus_switch(
+        let switched_degree = modulus_switch(
             monomial_degree,
-            polynomial_size,
-            ModulusSwitchOffset(0),
-            LutCountLog(0),
+            CiphertextModulusLog(polynomial_size.log2().0 + 1),
```

Comment:
as discussed let's have a small wrapper pbs_modulus_switch which does that computation for the person calling it

Reply:
Done